### ISSUE TITLE
Installer check for .NET 6 Console

### ISSUE THREAD

**Issue by jr551** [crowdsecurity/cs-windows-firewall-bouncer#5]
The installer needs to warn if Download .NET 6.0 Runtime (for console apps) is not installed, because if it is not installed then the installer gives a random warning about the service cannot start

**Comment by rebornnetwork** [comment]
Agreed. been having this issue and only checking here did i find the way to resolve this!
